fearless/Modules/SoraCard/PhoneVerificationCode/PhoneVerificationCodeInteractor.swift

```diff
@@ -113,18 +115,36 @@ extension PhoneVerificationCodeInteractor: SignInWithPhoneNumberVerifyOtpCallbac
 
     func onVerificationFailed() {
         codeState = .wrong("Incorrect or expired OTP")
+        output?.didReceive(state: codeState)
     }
 
     func onSignInSuccessful(refreshToken: String, accessToken: String, accessTokenExpirationTime: Int64) {
         let token = SCToken(refreshToken: refreshToken, accessToken: accessToken, accessTokenExpirationTime: accessTokenExpirationTime)
         SCTokenHolder.shared.set(token: token)
 
-        service.getUserData(callback: GetUserDataCallback())
+        service.getUserData(callback: getUserDataCallback)
         codeState = .succeed
-        output?.didReceiveSignInSuccessfulStep(data: data)
     }
 
     func onError(error: PayWingsOAuthSDK.OAuthErrorCode, errorMessage: String?) {
         codeState = .wrong(errorMessage ?? error.description)
     }
 }
+
+extension PhoneVerificationCodeInteractor: GetUserDataCallbackDelegate {
+    func onUserData(
+        userId: String,
+        firstName: String?,
+        lastName: String?,
+        email: String?,
+        emailConfirmed _: Bool,
+        phoneNumber _: String?
```

Comment:
please check unused vars

Reply:
It's okay, because it's library method, but we don't need all of this parameters.